examples/ConcordiumNetSdk.Examples.Transactions.RegisterData/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using ConcordiumNetSdk.Types;
+using ConcordiumNetSdk.Transactions;
+using ConcordiumNetSdk.Wallets;
+using ConcordiumNetSdk.Client;
+
+using Grpc.Core;
+
+namespace ConcordiumNetSdk.Examples.Transactions;
+
+/// <summary>
+/// Example demonstrating how to submit a register data transaction.
+///
+/// The example assumes you have your account key information stored
+/// in the Concordium browser wallet key export format, and that a path
+/// pointing to it is supplied to it from the command line.
+///
+/// Cfr. <see cref="RegisterDataTransactionExampleOptions"/> for more
+/// info on how to run the program, or refer to the help message.
+/// </summary>
+class Program
+{
+    static void SendRegisterDataExample(RegisterDataTransactionExampleOptions options)
+    {
+        // Read the account keys from a file.
+        WalletAccount account = WalletAccount.FromBrowserWalletExportFormat(options.WalletKeysFile);
```

Comment:
These are not tested, and I just noticed that these examples currently do not read the file, but pass the specified path instead. I'll fix it and test them before requesting re-review and merging.